packages/hash/frontend/src/components/GlideGlid/use-grid-tooltip.tsx

```diff
@@ -0,0 +1,111 @@
+import { DataEditorRef } from "@glideapps/glide-data-grid";
+import { Popover } from "@hashintel/hash-design-system";
+import { PopoverPosition, Typography } from "@mui/material";
+import _ from "lodash";
+import { bindPopover, usePopupState } from "material-ui-popup-state/hooks";
+import { RefObject, useCallback, useState } from "react";
+import { useWindowEventListener } from "rooks";
+import { GridTooltipManager } from "./use-grid-tooltip/grid-tooltip-manager";
+import {
+  GridTooltip,
+  TooltipCellProps,
+  UseGridTooltipResponse,
+} from "./use-grid-tooltip/types";
+
+export const useGridTooltip = (
+  gridRef: RefObject<DataEditorRef>,
+): UseGridTooltipResponse => {
+  const popupState = usePopupState({
+    variant: "popover",
+    popupId: "grid-tooltip",
+  });
+
+  // prevents tooltip to stay at cursor position while user scrolls vertically
```

Comment:
I'm not sure this makes sense grammatically – could you rephrase?

Reply:
Rephrased as `prevent tooltip from staying at the same position when user scrolls vertically`